src/Mono.Android/Android.Hardware/HardwareBufferUsage.cs

```diff
@@ -0,0 +1,63 @@
+namespace Android.Hardware;
+
+[global::System.Runtime.Versioning.SupportedOSPlatformAttribute ("android26.0")]
+public enum HardwareBufferUsage : long
```

Comment:
@jpobst: The values of these enum members look rather "flags-like"; should this be a `[Flags]` enum?